### DISCUSSION TITLE
No OData topics for '.NET Conf' next month?

### DISCUSSION THREAD

**Discussion by julealgon** [OData/AspNetCoreOData#721] [General]
Are there any plans to showcase OData stuff in the upcoming .NET Conf event? Looking at the event page and presenters, I don't immediately see anything that brings OData into the table.

Have you guys thought about that? @xuzhg / @habbes / @mikepizzo 

The OData stack suffers from being extremely poorly advertised I think, and this could be such an interesting opportunity to bring it to the forefront, especially with the improvements made on v8. This is such a powerful framework/protocol, having something over there could bring a ton of awareness to it.

Thoughts? Is it just too late to bring this up I wonder?

**Comment by xuzhg** [comment]
Early this month, we had an OData session at .NET @Conf 2022. 
Mike made the overall OData introduction, I showed OData functionalities using an ASP.NET Core OData app. Javier showed a little bit OData client and Blazor E2E and Swagger/OpenApi.

I think you can find the session through .NET @Conf link.

**Reply by julealgon** [reply]
Wait.... I'm confused now Sam. Isn't the .NET Conf 2022 happening next month? Are you talking about some other event? Can you provide a link please?
